### PR TITLE
Clarifying use of allocate with allow_primary

### PR DESCRIPTION
**Pull request by nellicus**
Clarifying use of allocate with allow_primary

### CODE REVIEW

**Comment by nellicus** [comment]
@ESamir @jjfalling @clintongormley can anyone review please?

**Comment by jjfalling** [comment]
Looks good

File: docs/reference/cluster/reroute.asciidoc [hunk 1]
```diff
@@ -68,3 +68,5 @@ The commands supported are:
     allocate the shard to. It also accepts `allow_primary` flag to
     explicitly specify that it is allowed to explicitly allocate a primary
     shard (might result in data loss).
+    
+WARNING: Using `allow_primary` in conjunction with `allocate` command to force a primary shard allocation, will cause data contained in existing targeted shard to be lost. Use with caution.
```
Comment (clintongormley) [inline docs/reference/cluster/reroute.asciidoc]: How about:

    The `allow_primary` parameter will force a new _empty_primary shard to be allocated, without
    any data. If a node rejoins the cluster later on, and has a copy of the original primary shard,
    including data, that data will be deleted: the old shard copy will be replaced by the new
    live shard copy.

(still not perfect)

Comment (nik9000) [inline docs/reference/cluster/reroute.asciidoc]: I do like Clint's wording better. If he had fewer commas I'd merge his wording.

Comment (jjfalling) [inline docs/reference/cluster/reroute.asciidoc]: Do you think the warning tag is too much? We've seen people delete their
data multiple times not realizing that flag can be destructive.

Comment (clintongormley) [inline docs/reference/cluster/reroute.asciidoc]: No, I absolutely think it should be a WARNING.

**Comment by clintongormley** [comment]
Rephrased without so many commas and merged.

**Pull request closed without merging**